openedx_webhooks/tasks/github.py

```diff
@@ -413,17 +416,27 @@ def github_community_pr_comment(pull_request, jira_issue, people=None):
     people = {user.lower(): values for user, values in people.items()}
     pr_author = pull_request["user"]["login"].decode('utf-8').lower()
     created_at = parse_date(pull_request["created_at"]).replace(tzinfo=None)
+    missing_committers = []
+    if commits:
```

Comment:
Is this to check for empty commits, or commits == None? If it's only for empty, then you can just let the for loop take care of it.